internal/webdavmount/webdavmount.go

```diff
@@ -110,27 +110,30 @@ type webdavDir struct {
 var symlinksAreUnsupportedLogged = new(int32)
 
 func (d *webdavDir) Readdir(n int) ([]os.FileInfo, error) {
```

Comment:
btw, now that I'm looking at this function I think it's completely incorrect - it will truncate directory and never allow pagination, no need to fix it now, but can you put a TODO/file a bug to fix it?